engine/src/flutter/impeller/display_list/dl_dispatcher.cc

```diff
@@ -642,10 +642,23 @@ void DlDispatcherBase::drawArc(const DlRect& oval_bounds,
                                bool use_center) {
   AUTO_DEPTH_WATCHER(1u);
 
-  PathBuilder builder;
-  builder.AddArc(oval_bounds, Degrees(start_degrees), Degrees(sweep_degrees),
-                 use_center);
-  GetCanvas().DrawPath(builder.TakePath(), paint_);
+  if (paint_.stroke_width >
```

Comment:
Maybe add a breadcrumb back to the original issue here, or a comment explaining why we're doing this conversion.